delegation/tasks/3-in-progress/ADV-0032-resolver-model-priority.md

```
@@ -0,0 +1,117 @@
+                return (config.model, config.api_key_env)
+            raise
+
+    # Legacy only
+    if config.model:
+        return (config.model, config.api_key_env)
+
+    raise ResolutionError("No model or model_requirement specified")
+```
+
+**New code**:
+```python
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)